### PR TITLE
[#950] Use surge as alternative for netlify

### LINKED ISSUE #950: Find a netlify alternative
Netlify is implementing a limit of 300 build minutes per month; it may not be sufficient for RepoSense as we have been using it as a PR review tool.
Hence, we might need to look out for an alternative.

https://www.netlify.com/pricing/#features

### PR DESCRIPTION
**Pull request by jamessspanggg**
Fixes #950

```
Netlify's build quota is not enough for RepoSense's build usage.

Let's deploy on Surge instead, using Travis to run the build and provide
a Github Status on each PR that links users to the deployed Surge site.
```

### CODE REVIEW

**Comment by jamessspanggg** [comment]
@eugenepeh the github status is working now. Thanks!

File: .travis.yml [hunk 1]
```diff
@@ -60,3 +57,7 @@ cache:
 
 after_success:
   - bash <(curl -s https://codecov.io/bash)
+  - git fetch --all && git config --global user.email "-" && git config --global user.name "-"
```
Comment (jamessspanggg) [inline .travis.yml]: @eugenepeh may i get the purpose of `git config --global user.email "-" && git config --global user.name "-"` in the run command?

Comment (eugenepeh) [reply .travis.yml]: There are some git commands that cannot be performed without the user.name and user.email value. We simply use the `-` value to bypass that need.

Comment (jamessspanggg) [reply .travis.yml]: ahh ok, noted.

**Comment by fzdy1914** [comment]
Can I propose your commit message? I will merge the PR now.

**Comment by jamessspanggg** [comment]
> Can I propose your commit message? I will merge the PR now.

done

**Comment by fzdy1914** [comment]
The travis fails, should we only run the surge for one sdk only?

**Comment by jamessspanggg** [comment]
> The travis fails, should we only run the surge for one sdk only?

hmm, but it may be the case that the first sdk fails and the second is successful right. In this way we still get a preview, if we run both.

**Comment by eugenepeh** [comment]
>hmm, but it may be the case that the first sdk fails and the second is successful right. In this way we still get a preview, if we run both.

I think its fine if we do the preview in just one env to slim down the build time; either way passing the CI test is a requirement for the PR to be merged.

**Comment by jamessspanggg** [comment]
@eugenepeh @fzdy1914 updated to only run on the linux build.

**Pull request merged**